WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Free to Paid Plans/FreeToPaidPlansDashboardCardHelper.swift

```diff
@@ -12,15 +12,7 @@ import Foundation
             return false
         }
 
-        /// If this propery is empty, it indicates that domain information is not yet loaded
-        let hasLoadedDomains = blog.domains?.isEmpty == false
-        let hasMappedDomain = blog.hasMappedDomain()
-        let hasFreePlan = !blog.hasPaidPlan
-
-        return blog.supports(.domains)
-            && hasFreePlan
-            && hasLoadedDomains
-            && !hasMappedDomain
+        return blog.supports(.domains) && !blog.hasPaidPlan
```

Comment:
The cards are visible for free domain sites on my tests. 🟢 
I have compared the logic with the [Android conditions](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/plans/PlansCardUtils.kt#L40-L45). All look same.
- Do we check `isAdmin` somewhere?

Reply:
Yes.

`blog.supports(.domains)` checks:

```
return ([self isHostedAtWPcom] || [self isAtomic]) && [self isAdmin] && ![self isWPForTeams];
```